tests/test_trading_model.py

```diff
@@ -0,0 +1,171 @@
+import pytest
+from django.contrib.auth.models import User
+from stocks.models import InvestmentBucket, Stock
+from stocks.historical import create_stock
+from django.db.models.signals import post_save
+from yahoo_historical import Fetcher
+from unittest import mock
+
+
+def setup_module(module):
+    """
+    Mock out any externals
+    """
+    post_save.disconnect(receiver=create_stock, sender=Stock)
+    module.original_init_method = Fetcher.__init__
+    module.original_getHistorical_method = Fetcher.getHistorical
+    Fetcher.__init__ = mock.Mock(return_value=None)
+    Fetcher.getHistorical = mock.Mock(return_value=None)
+
+
+def teardown_module(module):
+    """
+    Restore externals
+    """
+    Fetcher.__init__ = module.original_init_method
+    Fetcher.getHistorical = module.original_getHistorical_method
+
+
+# @pytest.mark.django_db(transaction=True)
+# def test_trading_account_total_value():
+#     user = User.objects.create(username='christophe', password="iscool")
+#     trading_account = user.profile.trading_accounts.create(
+#         account_name="spesh"
+#     )
+"""Total value is currently not defined"""
+""" Have to write an available cash test, after fixing
+available cash (current implementation is incorrect) """
```

Comment:
Let's just delete that method completely, since it doesn't make as much sense on trading_account as I thought